pom.xml

```diff
@@ -45,28 +45,6 @@
             <artifactId>jopt-simple</artifactId>
             <version>5.0.4</version>
         </dependency>
-        <!-- JAXB dependencies added for some java versions which
```

Comment:
This JAXB just causes us problems. It is used only at one place, and even at that one place I was able to use an already-included dependency (Bouncycastle).